Makefile

```diff
@@ -40,7 +40,7 @@ dev:
 
 # test runs the tests
 test:
-	@go test -timeout=60s -parallel=10 ./...
+	@go test -timeout=240s ./... $(TESTARGS)
```

Comment:
The tests took ~160s to finish on my machine without the `t.Parallel()` calls.

Reply:
I removed the `t.Parallel()` calls because we hit rate limits every time we run these tests. We run the tests infrequently (usually only happens when we release). Also, this plugin in low traffic for us, so there isn't a large cost to waiting a bit longer for the tests to complete.